wandb/sdk/lib/server.py

```diff
@@ -36,3 +37,20 @@ def query_with_timeout(self, timeout: int | float = 5) -> None:
         # TODO(jhr): should we kill the thread?
         self._viewer, self._serverinfo = viewer_tuple
         self._flags = json.loads(self._viewer.get("flags", "{}"))
+
+    @property
+    def viewer(self) -> dict[str, Any]:
+        """
+        Returns the viewer information queried from the backend `viewer` GraphQL API.
+
+        The viewer contains information about the currently authenticated user, including:
+        see the GraphQL viewer query defined here:
+        https://github.com/wandb/wandb/blob/230213f8bb6c9ad0c9736e82b00a23e84e73cfae/wandb/sdk/internal/internal_api.py#L948-L966
+
+        Returns:
+            dict: The viewer information dictionary, or empty dict if unavailable
+        """
```

Comment:
The docstring summary line needs to be attached to the `"""` in the style that we use. A `Returns` section isn't needed if the summary line starts with the word `Returns`. The entire line should also fit in 80 characters.

```py
@property
def viewer(self) -> dict[str, Any]:
    """Returns information about the currently authenticated user.

    If this instance was created with a valid API key, this includes <what>.

    If the API key is not valid <any other reasons?>, returns an empty dict.
    """
```